test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -227,21 +226,17 @@ class ClientLbEnd2endTest : public ::testing::Test {
     // Workaround Apple CFStream bug
     gpr_setenv("grpc_cfstream", "0");
 #endif
+    grpc_init();
```

Comment:
Do you still need the init/shutdown_blocking pair at all?

Reply:
I think so. We'll need to prevent an actual shutdown from happening by holding a grpc_init. Otherwise the registered lb policy factory might get lost after test cases complete.

Reply:
It sounds like it should go to the SetupTestCase of the actual test class rather than the base class?

Reply:
That's a good point.